src/auth/jwtService.ts

```diff
@@ -0,0 +1,41 @@
+import * as jwt from 'jsonwebtoken';
+import { Exception } from '../exception';
+import { injectJWTConfig } from './injections';
+import { IJWTConfig } from './jwtConfig';
+
+export class jwtService {
+    private jwtConfig: IJWTConfig;
+
+    public constructor() {
+        this.jwtConfig = injectJWTConfig();
+    }
+
+    public createJWT = (payload: any) => {
+        try {
+            const token = jwt.sign(payload, this.jwtConfig.secretKey, { expiresIn: this.jwtConfig.expiresIn, issuer: this.jwtConfig.issuer })
+            return {
```

Comment:
i don't think that we need to return information about expiration time as it's already in token